File: C/M_Replace_MinMax.c
#include<stdio.h>
int main ()
{
    int n;
    scanf("%d", &n);

    int arr[n];
    int a;
    for(int i=0; i<n; ++i) {
        scanf("%d ", &arr[i]);
    }

    int max= -100000, min=100000;
    for(int i=0; i<n; ++i) {
        if(max<arr[i]) {
            max = arr[i];
        }

        if(min>arr[i]) {
            min = arr[i];
        }
    }



    for(int i=0; i<n; ++i) {
        if(arr[i]==max) {
            arr[i] = min;
        } else if (arr[i]==min) {
            arr[i] = max;
        }
    }


    for(int i=0; i<n; ++i) {
        printf("%d ", arr[i]);
    }

    return 0;
}

File: Python/M_Replace_MinMax.py
n = int(input())
arr = list(map(int, input().split()))
maximum = max(arr)
minimum = min(arr)
min_index = arr.index(minimum)
max_index = arr.index(maximum)
arr[max_index], arr[min_index] = arr[min_index], arr[max_index]
for i in arr:
    print(i, end=' ')